chellow/templates/base.html

```diff
@@ -126,8 +126,7 @@
     		<link rel="stylesheet" href="/style" />
 		    <link rel="stylesheet" href="/static/css/bootstrap.min.css" />
         	<link rel="stylesheet" href="/static/font-awesome-4.6.3/css/font-awesome.min.css" />
-        	<link rel="stylesheet" href="/chellowcss" />
-        	
+        	<link rel="stylesheet" href="{{context_path}}/chellowcss" />
```

Comment:
There's a {{context_path}} here, but we don't use it any more. Also, is there any reason why the stylesheet can't be in 'static'?